collections/_article/gdc-2025-retrospective.md

```diff
@@ -0,0 +1,74 @@
+
+But the Godot community shared the word and answered! We were stunned by the number of people that showed up.
+
+Those who attended the meetup had the opportunity to talk with regular Godot contributors, Emilio Coppola, and Ariel Manzur (co-founder of the Godot Engine and board member of the Godot Foundation).
+
+Our many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the two hour mark, somebody told me that the last people left nearly one hour after.
+
+## Showcased Godot games
+### Kneedle Drop: An Into the Woolderness Game
+
+"A RHYTHM KNITTING GAME?!" That's how [Immortal Funk Games](https://www.immortalfunkgames.com/) describes their spin-off game taylor-made (and chosen!) for the [alt.ctrl.GDC showcase of alternative controllers](https://gdconf.com/alt-ctrl-gdc).
+
+<img src="/storage/blog/gdc-2025/kneedle-drop-gameplay.webp" alt="Kneedle Drop gameplay, where two people hold each two controllers shaped as knitting needles." />
+
+In this game that mimics the essence of knitting as well as Guitar Hero does for playing guitar, either you tap the point of the needles, or you tap the left controller on the right one from one side or the other, all while keeping the rhythm.
+
+<img src="/storage/blog/gdc-2025/kneedle-drop-team.webp" alt="The team behind Kneedle Drop, in front of their game showing on a screen." />
+
+The team was explaining to me how nice working with Godot is, especially for artists.
+
+Be sure to check out [Into the Woolderness](https://www.immortalfunkgames.com/into-the-woolderness), the main project of the team. It's a cozy game about crafting and knitting garments for your dogs. Be reassured (or sad), as this one will play on a standard controller.
+
+### Year Unknown
+
+[Year Unknown](https://store.steampowered.com/app/2565520/Year_Unknown/) is a game released in October last year by Julian Heuser. It is a "walking simulator" about exploring a minimalist landscape with the help of two AIs, trying to uncover secrets about humanity.
+
+<img src="/storage/blog/gdc-2025/year-unknown.webp" alt="Julian Heuser before someone playing his game, Year Unknown." />
+
+The game was showcased at the [IGF](https://igf.com/) Pavillon as [a finalist for "Best Student Game" for the "2025 Independent Games Festival"](https://igf.com/finalists-and-winners). Even if the game didn't ultimately win the prize, few developers can actually say that they were nominated by the IGF. Congratulations, Julian!
+
+## Conclusion
+
+You know, as Godot is an international project and as its contributors usually work remotely, events like this one are usually the only way for us to actually meet. Meeting numerous colleagues, great corporate partners, and people that are so passionate about the project made the trip worthwhile.
```

Comment:
```suggestion
As Godot is an international project and its contributors mostly work remotely, events like this one are usually the only way for us to actually meet. Meeting numerous colleagues, great corporate partners, and people that are so passionate about the project made the trip worthwhile.
```
Changed to "mostly" to avoid close repetition of "usually".